scripts/setupvars/setupvars.sh

```diff
@@ -103,6 +103,11 @@ if [ -e "$INTEL_OPENVINO_DIR/deployment_tools/tools/post_training_optimization_t
     export PYTHONPATH="$INTEL_OPENVINO_DIR/deployment_tools/tools/post_training_optimization_toolkit:$PYTHONPATH"
 fi
 
+if [ -e "$INTEL_OPENVINO_DIR/deployment_tools/tests" ]; then
+    export MO_ROOT="$INTEL_OPENVINO_DIR/deployment_tools/model_optimizer"
+    export PYTHONPATH="$INTEL_OPENVINO_DIR/deployment_tools:$MO_ROOT:$PYTHONPATH"
```

Comment:
Why tests need `deployment_tools` to be available in PYTHONPATH? Perhaps need to add `deployment_tools/tests`?